src/_data/pricingFeatures.yaml

```diff
@@ -18,127 +23,122 @@ sections:
       - id: pipelines
         label: DevOps Pipelines
-        cloudValues: [ null, 'check', 'check' ]
+        cloudValues: [ null, null, 'check', 'check' ]
         selfHostedValues: [ null, 'check' ]
         tags: [ 'cloud', 'self-hosted' ]
         info: "<p>DevOps pipelines allows for you to setup different environments for development, testing and production Node-RED instances.</p>"
       - id: project-nodes
         label: Seamless Data Stream
-        cloudValues: [ null, 'check', 'check' ]
+        cloudValues: [ null, null, 'check', 'check' ]
         selfHostedValues: [ null, 'check' ]
         tags: [ 'cloud', 'self-hosted' ]
         info: "<p><a href='https://github.com/FlowFuse/nr-project-nodes' target='blank'>FlowFuse Project Nodes</a> enable the passing of data and messages between your Node-RED projects, included with all FlowFuse Projects by default.</p>"
       - id: device-groups
         label: Device Group Management
-        cloudValues: [ null, null, 'check' ]
+        cloudValues: [ null, null, null, 'check' ]
         selfHostedValues: [ null, 'check' ]    
         tags: [ 'cloud', 'self-hosted' ]
         info: "<p>Device Groups enables users to logically group their devices assigned to an application. Additionally, Device Groups can be integrated into the DevOps Pipeline.</p>"    
       - id: ha
         label: High Availability
-        cloudValues: [ null, null, 'check' ]
+        cloudValues: [ null, null, null, 'check' ]
         selfHostedValues: [ null, 'check' ]    
         tags: [ 'cloud', 'self-hosted' ]
         info: "<p>Leverage horizontal scaling for reliable and scalible processing of your data through Node-RED.</p>"      
   - label: Secure
     rows:
       - id: audit-log
         label: Audit Log
-        cloudValues: [ 'check', 'check', 'check' ]
+        cloudValues: [ 'check', 'check', 'check', 'check' ]
         selfHostedValues: [ 'check', 'check' ]
         tags: [ 'cloud', 'self-hosted' ]
         info: "<p>Keep track of everything going on in your Node-RED instances and FlowFuse.</p><p>Audit Logs provide details on what actions have taken place, when they happened, and who did them.</p>"
       - id: rbac
         label: Role-Based Access Control
-        cloudValues: [ 'check', 'check', 'check' ]
+        cloudValues: [ null, 'check', 'check', 'check' ]
         selfHostedValues: [ 'check', 'check' ]
         tags: [ 'cloud', 'self-hosted' ]
         info: "<p>FlowFuse's intuitive team management tooling makes it easy for you to control whom has access to what.</p>"
       - id: security
         label: Endpoint Security
-        cloudValues: [ 'None<br>HTTP Basic Auth', 'None<br>HTTP Basic Auth<br>FlowFuse Authentication', 'None<br>HTTP Basic Auth<br>FlowFuse Authentication' ]
+        cloudValues: [ null, 'None<br>HTTP Basic Auth', 'None<br>HTTP Basic Auth<br>FlowFuse Authentication', 'None<br>HTTP Basic Auth<br>FlowFuse Authentication' ]

```

Comment:
The instance do have authentication set up? Cc @knolleary

Reply:
@ZJvandeWeg Freemium is Devices only; this row relates to hosted instances

Reply:
So the label is incorrect? I know it's edge instance only, but it's not clear?
Now it seems like we're not applying endpoint security.

Reply:
I've just revised the popup to clarify that this row is about hosted instances.